rest_api/aggregation/database_hooks.py

```diff
@@ -121,6 +123,35 @@ def _expression(self, species_contam):
         return sorted(k for k, v in species_contam['contaminant_unique_mapped'].items() if v > 500)
 
 
+class RequiredYield(Calculation):
+    @cached_property
+    def quantised_yields(self):
+        return cfg['available_yields']
+
+    def _expression(self, genome_size):
+        return {
+            # keys have to be strings to be valid BSON
+            str(required_coverage) + 'X': self._get_yield(genome_size, required_coverage)
+            for required_coverage in cfg['available_coverages']
+        }
+
+    def _get_yield(self, genome_size, required_coverage):
+        exact_yield = genome_size * required_coverage
```

Comment:
This calculation makes assumption about the genome size/ exact yield unit. I'm assuming they are both in Gb. W should make it explicit if this is the case via a comment at least